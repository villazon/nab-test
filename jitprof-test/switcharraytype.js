var a = [1,2,3];
a[3]="a";

var c = ["1","2"];
c[2]=1;
