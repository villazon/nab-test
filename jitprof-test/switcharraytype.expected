---------------------------
Report of switching array type
 * [location: (jitprof-test/switcharraytype.js:2:1:2:5)] <- No. usages: 1
...
Number of switching array type spotted: 1
[****]SwitchArrayType: 1
